ansible/parted.yml

```diff
@@ -0,0 +1,13 @@
+- hosts: cluster
+  tasks:
+  - name: Create partitions
```

Comment:
Might be helpful if this was more description. For example Create partitions for BeeGFS

Reply:
In theory, this can be used for creating any partition, not just BeeGFS

Reply:
But point taken on making it more verbose...